nightingale_msgs/action/PoseEstimation.action

```diff
@@ -0,0 +1,10 @@
+# input is the body part as a string
+# {"body", "head"}
+string target
+---
+# returns a 3D coordinate PointStamped
+# of the bin goal in base link frame
+geometry_msgs/PointStamped bin_goal
+---
+# returns if issue getting pose or accurate coords 
+int8 pose_detect_failed
```

Comment:
@FelixKHWong  this is the feedback part of ur action. not the return part. the idea behind feedback, is that its continuously published. so for RR, the feedback is the distance to the goal. we constantly send that as feedback. 

do u continuous publish whether the pose_detected_failed while the aciton is running? if not, then this should be part of the result.

Reply:
oh I see. Since the node returns a lot quicker than I thought(could be a service) I'll remove this and we will just treat the perception node as a service(even though it's an action) because it is already implemented as such